node/mock/txFeeHandlerStub.go

```diff
@@ -32,7 +32,7 @@ func (fhs *FeeHandlerStub) DeveloperPercentage() float64 {
 }
 
 // MaxGasLimitPerBlock -
-func (fhs *FeeHandlerStub) MaxGasLimitPerBlock() uint64 {
+func (fhs *FeeHandlerStub) MaxGasLimitPerBlock(uint32) uint64 {
 	return fhs.MaxGasLimitPerBlockCalled()
```

Comment:
change the mock method signature also

Reply:
it is not used rigth now - in any test it is only one value. there is no need.